libgloss/arc/hl/Makefile.in

```diff
@@ -0,0 +1,81 @@
+DESTDIR =
```

Comment:
why don;t u use the top `Makefile.in` ?

Reply:
We already have subdir `dw_uart` here, so I think we should use the same pattern here.

Reply:
If something is done in one way it does mean it should be the rule :), but if you still want to use your own Makefile.in please check how is done in ARM case, i.e., without configure.

Reply:
Done, thanks for pointing to this example. Looks much simpler now.